elasticsearch/Dockerfile

```diff
@@ -1,6 +1,8 @@
-FROM docker.elastic.co/elasticsearch/elasticsearch:8.5.2
-
+FROM docker.elastic.co/elasticsearch/elasticsearch:8.5.3
+COPY querqy-elasticsearch-1.8.es853.0-SNAPSHOT.zip /usr/share/elasticsearch/querqy-elasticsearch-1.8.es853.0-SNAPSHOT.zip
```

Comment:
does the snapshot zip file need to be checked into the repo? There is no maven snapshot release that can be pulled down maybe?